src/editor/Objects/ExtrudeController.tsx

```diff
@@ -0,0 +1,47 @@
+import { useRef, useState } from 'react';
+import * as THREE from 'three';
+import { ThreeElements, useFrame } from '@react-three/fiber';
+
+function ExtrudeController(props: ThreeElements['mesh']) {
+  const ref = useRef<THREE.Mesh>(null!);
+  const [hovered, hover] = useState(false);
+  const [clicked, click] = useState(false);
+
+  const shape = new THREE.Shape();
+  shape.moveTo(0, 0);
+  shape.lineTo(1, 0);
+  shape.lineTo(1, 1);
+  shape.lineTo(0, 1);
+  shape.lineTo(0, 0);
```

Comment:
nad tym ksztaltem bedziemy musieli sie troche zastanowic. na razie niech to wejdzie, ale to jest jeden z tych trudniejszych ksztaltow, ktory wymaga punktow. w przyszlosci bedzie trzeba przemyslec UI do tego